packages/material-ui/src/styles/index.d.ts

```diff
@@ -1,11 +1,12 @@
 export * from './colorManipulator';
 export {
-  default as createMuiTheme,
+  default as createTheme,
   default as unstable_createMuiStrictModeTheme,
   ThemeOptions,
   Theme,
   Direction,
-} from './createMuiTheme';
+} from './createTheme';
+export { default as createMuiTheme } from './createMuiTheme';
```

Comment:
does this file still exist? Why not export from `createTheme` instead?

Reply:
I added a new file to keep compatible with users importing it directly (I know we don't recommend it).

```js
import createMuiTheme from "@material-ui/core/styles/createMuiTheme";
```

Reply:
We recommend imports with length at most of 3rd path, it's safe to remove it.

Reply:
> I added a new file to keep the compatibility with users importing it directly (I know we don't recommend it).

It's not only not recommended but not supported. It will likely break during v5.